docs/en/ufs/S3.md

```diff
@@ -380,6 +380,6 @@ for more details.
 ### Prevent Creating Zero-byte Files
 Alluxio creates zero-byte files in S3 as a performance optimization.
 For a bucket mounted with read-write access, zero-byte file creation (S3 PUT operation) is not
-restricted to writes using Alluxio but also occurs when listing contents of the under storage.
```

Comment:
i think we generally use `writes` for this. but if confusing this can be expanded to be `is not restricted to write operations using Alluxio`

Reply:
yes, changed accordingly